packaging/packager_deb.sh

```diff
@@ -5,46 +5,61 @@ readonly distro="${DISTRO:-ubuntu}"
 readonly distro_version="${VERSION:-xenial}"
 readonly distro_root="${ROOT:-$HOME}"
 
-# deb dependencies
+echo "------------- deb dependencies -------------"
 export DEBIAN_FRONTEND=noninteractive
 apt-get update
-# python3
-apt-get -y install python3 python3-coverage
-# python-imaging
-apt-get -y install python3-pil
-# pip dependencies (for dependencies not available as DEB)
+echo "------------- python3 install -------------"
+apt-get -y install python3 python3-venv
+echo "----- pip dependencies (for dependencies not available as DEB) -----"
 apt-get -y install gcc libx11-dev libxtst-dev python3-dev libpng-dev python3-pip
+
+echo "----- create a venv to use that as the python env (needed for python 3.12 compatibility specific to Ubuntu) -----"
+python3 -m venv docker_venv --system-site-packages
+source docker_venv/bin/activate
+
+echo "------------- upgrade pip -------------"
```

Comment:
In this script there are spaces but in the RPM script there are none. Can you please compare both scripts side by side (as half-screen editor windows) and make sure the formatting is identical?